actors/builtin/multisig/multisig_actor.go

```diff
@@ -303,13 +306,16 @@ func (a *MultiSigActor) _rtApproveTransactionOrAbort(rt Runtime, callerAddr addr
 			txn.Params,
 			txn.Value,
 		)
+		// TODO REVIEW if getPendingTxn returns the hamt node we can save a look up by passing it here can deleting.
```

Comment:
engrish: meant to say we can save a look up by passing the hamt node to the delete method for slightly faster look up when deleting.